security-controls/deployment.yaml

```diff
@@ -36,11 +37,19 @@ spec:
             envFrom:
             - secretRef:
                 name: ${SERVICE_SECRET}
+            - secretRef:
```

Comment:
Can we add these secrets in the same file with other `security-controls` namespace secrets?

Reply:
No need, that's harder to rotate should the need arise.

Reply:
Why keeping keys in a separate file will help with rotation?

Reply:
They're not really files, right? They're k8s secrets? It is necessary to know all the env vars within a secret in order to change any one of them because you appear to need to rewrite the whole secret. Maybe you have a simpler way -- I may have missed something in how to manage these.